assets/js/vue/FileManager.vue

```diff
@@ -354,6 +354,16 @@ const initDropzone = () => {
     maxFilesize: null,
     clickable: '#upload-file-button',
     timeout: 0,
+    uploadprogress(file) {
+      if (file.xhr.status === 204) {
+        if (myFileManager.$parent.totalFileSize
+          > (myFileManager.$parent.doneFileSize + file.upload.chunks[0].bytesSent)) {
+          myFileManager.$parent.doneFileSize += file.upload.chunks[0].bytesSent;
```

Comment:
```suggestion
        if (myFileManager.$parent.totalFileSize >= file.upload.bytesSent) {
          myFileManager.$parent.doneFileSize = file.upload.bytesSent;
```

Reply:
I tried it, bytesSent doesn't get updated quick enough. Not working as expected